test/ui_test_suite.rb

```diff
@@ -273,20 +273,22 @@ def teardown
     next_btn = @driver.find_element(:id, 'next-btn')
     next_btn.click
 
+    # TODO: figure out how to get this to work with ingest
     # upload a coordinate labels file
-    wait_for_render(:class, 'add-coordinate-labels')
-    add_coords_btn = @driver.find_element(:class, 'add-coordinate-labels')
-    add_coords_btn.click
-    wait_for_render(:class, 'initialize_labels_form')
-    coords_form = @driver.find_element(:class, 'initialize_labels_form')
-    cluster_select = coords_form.find_element(:id, 'study_file_options_cluster_group_id')
-    cluster_select.send_keys('Test Cluster 1')
-    upload_coords = coords_form.find_element(:class, 'upload-labels')
-    upload_coords.send_keys(@test_data_path + 'coordinate_labels_1.txt')
-    upload_btn = coords_form.find_element(:id, 'start-file-upload')
-    sleep(0.5)
-    upload_btn.click
-    close_modal('upload-success-modal')
+    # wait_for_render(:class, 'add-coordinate-labels')
```

Comment:
Does this branch of SCP not support upload coordinate label files?  If so, is such support critical for launch?

Reply:
It will support them, but there is an inherent race condition right now where the cluster object itself is not created quickly enough for the UI test to pass - we'd have to wait the minute or two for that parse to complete for this particular test to work.  In a real-world scenario, users will still be able to upload coordinate labels as always, so it shouldn't be a change in production.  It's just this particular test can't do it in a timeframe that I consider tractable.